web/app/js/components/ResourceDetail.jsx

```diff
@@ -380,15 +380,11 @@ export class ResourceDetailBase extends React.Component {
           isTcpTable={true}
           metrics={this.state.podMetrics} />
 
-        <Grid container direction="column" justify="center">
-          <Grid item>
```

Comment:
Another solution would be to add prop `xs={12}` to Grid item, but I think that removing these extra `Grids` is better.